opencraft/views.py

```diff
@@ -45,3 +53,46 @@ def get_redirect_url(self, *args, **kwargs):
             return reverse('instance:index')
         else:
             return reverse('registration:register')
+
+class HealthCheckView(View):
+    """
+    Health Check view
+    """
+    def get(self, request):
+        try:
+            # Verify that the postgres database backend is reachable
+            self._check_postgres_connection()
```

Comment:
@mavidser, will the app even be able to serve requests when the database is down? Just thinking out loud. CC @mtyaka

Reply:
This is acknowledged under "Author notes", but I think it would be good to also add a comment explaining that to the code.